README.md

```diff
@@ -48,32 +44,15 @@ Automatic playthrough of Dynamax Adventures to shiny-hunt legendaries.
 
 ## Get Me Started!
 
-Interested?
-
-**If you are new to automation**, start from with the [Microcontroller programs](https://github.com/PokemonAutomation/Microcontroller).
-
-**If you are coming from other projects such as AutoController or pokescripts.xyz**, we still recommend looking at the [Microcontroller programs](https://github.com/PokemonAutomation/Microcontroller) to familiarize yourself with our platform.
-
-Once you are comfortable with the microcontroller programs, you can proceed to the [Computer-Control programs](https://github.com/PokemonAutomation/ComputerControl) to unleash the full capability of our automation.
-
-**Program Lists:**
-- [Microcontroller Program List](https://github.com/PokemonAutomation/Microcontroller/blob/master/Wiki/Programs/README.md)
-- [Computer Control Program List](https://github.com/PokemonAutomation/ComputerControl/blob/master/Wiki/Programs/README.md)
-
-**Still not sure where to start?**, take a look at our [Decision Flow Chart](https://github.com/PokemonAutomation/About/blob/master/Decision%20Flow%20Chart.md).
-
-## Repos
-
-**Main Repos:**
-
-- [About:](https://github.com/PokemonAutomation/About) (this page)
-- [Microcontroller:](https://github.com/PokemonAutomation/Microcontroller) Repo for all microcontroller and Arduino-based automation.
-- [Computer-Control:](https://github.com/PokemonAutomation/ComputerControl) Repo for all computer-based automation.
-
-**Other Repos:**
+Interested? This is **PART 1**
 
-- [AutoMaxLair:](https://github.com/PokemonAutomation/AutoMaxLair) Stand-alone prototype for Pokémon Sword/Shield Dynamax Adventures.
+Choose the link below that aligns with your experience with electronics.
 
+| [Beginner](https://github.com/Ensamma/About/blob/master/HardwareBeginner.md) | [Experienced](https://github.com/Ensamma/About/blob/master/HardwareExperienced.md) |
+| --- | --- |
+| I have *not* used a microcontroller before | I have used a microcontroller before |
+| Make it work, simply | Options are okay |
+| I'm new to GitHub | I've used GitHub a few times |

```

Comment:
Let's add "I know nothing about circuits" and "I have a degree in electrical engineering." lol